app/login/index.php

```diff
@@ -137,6 +137,13 @@
 			# destroy session
 			$User->destroy_session();
 		}
+		
+		//check if SAML2 login is possible
+		$saml2settings=$Tools->fetch_object("usersAuthMethod", "type", "SAML2");
+		if($saml2settings!=false){
+			$Result->show("success", _('You can login with SAML2 <a href="'.$Tools->createURL().'/saml2">here</a>'));
```

Comment:
I think there's a minor fix here:
You should use `.create_link('saml2')` instead of raw `/saml2` for link generation